pf/tests/provider_checkconfig_test.go

```diff
@@ -667,7 +674,7 @@ func TestPreConfigureCallback(t *testing.T) {
 		  "response": {
 		    "inputs": {
 		      "version": "6.54.0",
-	              "configValue": "04da6b54-80e4-46f7-96ec-b56ff0331ba9"
+	              "configValue": "dd056dcd-154b-4c76-9bd3-c8f88648b5ff"
```

Comment:
This is now UnknownObjectValue = "dd056dcd-154b-4c76-9bd3-c8f88648b5ff" ... ever so slightly weird.